<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>626</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>994</width>
    <height>626</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>994</width>
    <height>626</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vista Inserisci Cliente</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>626</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>InserimentoNuovoCliente.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nome">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cognome">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>250</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="codiceFiscale">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataNascita">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="eta">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telefono">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>450</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color: lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_conferma">
   <property name="geometry">
    <rect>
     <x>468</x>
     <y>550</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>CONFERMA</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="InserimentoCliente.qrc"/>
 </resources>
 <connections/>
</ui>
